zoraxy/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: zoraxy_server_1
+      APP_PORT: 8000
+      PROXY_AUTH_ADD: "false"
+  server:
+    image: zoraxydocker/zoraxy:v3.1.2@sha256:38f23c5a507542bbd6eac0492bf6456be2004015594ca270a1a5998c8a377b74
+    restart: on-failure
+   #ports: -> The proxy service is listening on port 8400
+      # 8480:80
+      # 8443:443
+      # 8400:8000
+    volumes:
+      - ${APP_DATA_DIR}/config/:/opt/zoraxy/config/

```

Comment:
Let's do the same thing as the other submissions and put this one level deeper on the host inside a parent `data` directory so that it has similar structure to the majority of other apps, and it makes it easy to add more bind mounts inside `data` in the future.

```
- ${APP_DATA_DIR}/data/config/:/opt/zoraxy/config/
```

Reminder, you'll need to make the equivalent change to the committed directories.